File: Agenda_VB/Agenda_VB/Contacto.vb
Public Class Contacto
    Inherits Persona
    Private telefono As String
	Public Property Telefon() As String
		Get
			Return telefono
		End Get
		Set(ByVal Value As String)
			telefono = Value.Replace(" ", "").Replace("-", "")
		End Set
	End Property
	Private correo As String
	Public Property Corre() As String
		Get
			Return correo
		End Get
		Set(ByVal value As String)
			correo = value
		End Set
	End Property
	Public Sub New()
		MyBase.New()
		telefono = String.Empty
	End Sub
	Public Sub New(ByVal nombre As String, ByVal fechaNacimiento As DateTime?, ByVal telefono As String)
		MyBase.New()
		Me.telefono = telefono
	End Sub
	Public Overrides Function ToString() As String
		Return MyBase.ToString() + " " + telefono + " " + correo
	End Function
End Class

File: Agenda_C/Agenda_C/Contacto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Agenda_C
{
    internal class Contacto : Persona
    {
		private string numero;

		public string Numero
		{
			get { return numero; }
			set { numero = value; }
		}
        private string correo;

        public string Correo
        {
            get { return correo; }
            set { correo = value; }
        }

        public Contacto() : base()
        {
            numero = string.Empty;
            correo= string.Empty;
        }
        public Contacto(String nombre, DateTime? fecha, string telefono, string correo) : base(nombre, fecha)
        {
            this.numero = telefono;
            this.correo = correo;
        }
        public override string ToString()
        {
            return base.ToString() + " " + numero + " " + correo;
        }
    }
}
